# An array of anchors
Header header

Anchor[] anchors